lib/internal/http2/core.js

```diff
@@ -1255,6 +1256,12 @@ function connectionListener(socket) {
     socket.on('timeout', socketOnTimeout);
   }
 
+  // TLS ALPN fallback to HTTP/1.1
+  if ((options.allowHTTP1 === undefined || !!options.allowHTTP1 === true) &&
```

Comment:
Saw this pattern used elsewhere in Node.js core to coerce to Boolean. Thought it was a convenience to the user. Do you prefer to just do `options.allowHTTP1 === true`?